lib/web_ui/test/engine/semantics/semantics_helper_test.dart

```diff
@@ -142,26 +141,26 @@ void testMain() {
         expect(mobileSemanticsEnabler.semanticsActivationTimer, isNull);
 
         // Send a click off center
-        _placeholder!.dispatchEvent(html.MouseEvent(
+        _placeholder!.dispatchEvent(createDomMouseEvent(
           'click',
-          clientX: 0,
-          clientY: 0,
+          <Object?, Object?>{ 'clientX': 0,
```

Comment:
This spacing is kinda awkward. Could you maybe either do it all on one line, or do like
```
<Object?, Object?>{
  'clientX': 0,
  'clientY": 0,
}
```